app/src/pages/inside/logsPage/logItemInfo/logItemInfoTabs/attachments/index.js

```diff
@@ -0,0 +1,3 @@
+export { Attachments } from './attachments';
+export { AttachmentsContainer } from './attachmentsContainer';
```

Comment:
`export { AttachmentsContainer as Attachments } from './attachmentsContainer'`

The idea here is that we use only connected component outside. Presentational layer is needed only for testing/storybook in this case. That's why we don't want to export it from here.